src/index.html

```diff
@@ -855,9 +855,11 @@ <h4 class="modal-title" data-bind="locale: 'create_support_case'"></h4>
     <script src="vendors/cryptojs/rollups/aes.js"></script>
 
     <!-- bitcore -->
-    <script src="vendors/bitcore/browser/bundle.js"></script>
+    <script src="vendors/bitcore/bitcore-lib.js"></script>
```

Comment:
this might have to be changed to `vendors/bitcore/bitcore.js` when bitcore builds a new version ... (their builds are funky)